packages/expo-barcode-scanner/android/src/main/java/expo/modules/barcodescanner/utils/Helper.java

```diff
@@ -0,0 +1,51 @@
+package expo.modules.barcodescanner.utils;
+
+import android.os.Bundle;
+import android.util.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Helper {
```

Comment:
`BarCodeScannerHelper` maybe? All the other helpers have meaningful prefixes:
- https://github.com/expo/expo/blob/master/packages/expo-camera/android/src/main/java/expo/modules/camera/CameraViewHelper.java
- https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/ReactAndroid/src/main/java/com/facebook/react/views/scroll/VelocityHelper.java
https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/ReactAndroid/src/main/java/com/facebook/react/uimanager/TransformHelper.java